tests/models/test_sync_batchnorm.py

```diff
@@ -107,6 +108,7 @@ def test_sync_batchnorm_ddp(tmpdir):
         sync_batchnorm=True,
         num_sanity_val_steps=0,
         replace_sampler_ddp=False,
+        plugins=[DDPPlugin(find_unused_parameters=True)]
```

Comment:
had to set this here to true, as mentioned by @SeanNaren in the comment above. 
turns out this modification to the test is already in master branch

Reply:
Yeah for a test this is fine, but for the user you can't enable this unless you pass your own plugin, which is why I don't like this

Reply:
yes, it's also not agnostic to command line input / device without directly. A new trainer flag is probably best for user.